src/commands/lambda/constants.ts

```diff
@@ -96,6 +107,11 @@ export const VERSION_ENV_VAR = 'DD_VERSION'
 export const ENVIRONMENT_ENV_VAR = 'DD_ENV'
 export const EXTRA_TAGS_ENV_VAR = 'DD_TAGS'
 export const CAPTURE_LAMBDA_PAYLOAD_ENV_VAR = 'DD_CAPTURE_LAMBDA_PAYLOAD'
+export const ENABLE_PROFILING = 'CORECLR_ENABLE_PROFILING'
```

Comment:
Nit: All of the other env var consts end in `_ENV_VAR`, I would follow that pattern to be consistent